src/ModelAnalyzer.jl

```diff
@@ -41,9 +43,11 @@ be a subtype of `AbstractIssue`). In the verbose case it will provide a text
 explaning the issue. In the non-verbose case it will provide just the issue
 name.
 
-    summarize([io::IO,] issue::AbstractIssue; verbose = true)
+    summarize([io::IO,] issue::AbstractIssue; name_source = nothing, verbose = true)
 
 This variant allows summarizing a single issue instance of type `AbstractIssue`.
+The model tha led to the issue can be provided to `name_source`, it will be used
```

Comment:
```suggestion

The model that led to the issue can be provided to `name_source`, it will be used
```

Reply:
Why not just `; model::Union{Nothing,JuMP.AbstractModel} = nothing`?

Reply:
Fixed!
I don't want to type this because I want to have JuMP as a weakdep